include/lsst/ctrl/events/EventSystem.h

```diff
@@ -84,26 +77,23 @@ class EventSystem {
     Event* receiveEvent(const std::string& topicName);
     Event* receiveEvent(const std::string& topicName, const long timeout);
 
-    int64_t createOriginatorId();
-    int extractIPId(int64_t identificationId);
-    int extractProcessId(int64_t identificationId);
-    short extractLocalId(int64_t identificationId);
+    LocationID *createOriginatorId();
 
     StatusEvent* castToStatusEvent(Event* event);
     CommandEvent* castToCommandEvent(Event* event);
 
 private:
+    static EventSystem *defaultEventSystem;
```

Comment:
In this case, yes.   We've had use cases where configuration of event transmitters and receivers was done at initialization time, and then many layers of code later they were invoked.  One place this was used was previously was in pex_harness.  This is also used for status messages (not logging) for visualization of activity in alert production.